src/main/java/com/minecraftabnormals/abnormals_core/core/events/listener/ACEvents.java

```diff
@@ -1,35 +1,49 @@
-package com.minecraftabnormals.abnormals_core.core.events;
+package com.minecraftabnormals.abnormals_core.core.events.listener;
 
 import com.minecraftabnormals.abnormals_core.core.AbnormalsCore;
 import com.minecraftabnormals.abnormals_core.core.api.IAgeableEntity;
 import com.minecraftabnormals.abnormals_core.core.config.ACConfig;
+import com.minecraftabnormals.abnormals_core.core.util.DataUtil;
+import com.minecraftabnormals.abnormals_core.core.util.NetworkUtil;
+import net.minecraft.block.BlockState;
 import net.minecraft.entity.Entity;
 import net.minecraft.entity.LivingEntity;
 import net.minecraft.entity.player.PlayerEntity;
 import net.minecraft.item.ItemStack;
 import net.minecraft.item.Items;
 import net.minecraft.nbt.CompoundNBT;
+import net.minecraft.particles.ParticleTypes;
 import net.minecraft.potion.EffectInstance;
 import net.minecraft.potion.Effects;
 import net.minecraft.util.ActionResultType;
+import net.minecraft.util.Direction;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.SoundCategory;
+import net.minecraft.util.SoundEvent;
 import net.minecraft.util.SoundEvents;
+import net.minecraft.util.math.BlockPos;
 import net.minecraftforge.event.entity.living.LivingEvent;
 import net.minecraftforge.event.entity.player.PlayerInteractEvent;
+import net.minecraftforge.event.world.NoteBlockEvent;
+import net.minecraftforge.eventbus.api.EventPriority;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 import net.minecraftforge.fml.ModList;
 import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.registries.ForgeRegistries;
+
+import java.util.ArrayList;
+import java.util.List;
 
 /**
- * Events for mod compatibility.
- *
  * @author abigailfails
  */
 @Mod.EventBusSubscriber(modid = AbnormalsCore.MODID)
-public final class CompatEvents {
+public final class ACEvents {
```

Comment:
Changing this class' name is a breaking change.

Reply:
Reverted.